### DISCUSSION TITLE
신홍규 - 엔터테인먼트 - 음악 청취 습관 분석

### DISCUSSION THREAD

**Discussion by ShinHongKyu** [dinnercoding/wanted-pre-onboarding-challenge-data#110] [2023년 11월 프리온보딩 데이터 챌린지]
### 도메인 선정 이유
* 오랜 시간 음악 청취를 즐기며 본인의 청취 습관과 취향이 바뀌었을지 궁금하여 해당 도메인을 선정함

### 문제 정의
* 음악 청취 습관, 취향에 변화가 있었을지 파악하고자 한다.
* 특히 많이 듣는(반복재생을 많이 하는) 음악에 변화가 있을지 알아보고자 한다.
* 가능하다면 즐겨 듣는 음악들의 특징을 분석하고자 한다.

### 데이터 업로드
* 본인 스포티파이 계정 데이터 (스트리밍 기록, 플레이리스트, 라이브러리 등)
*  https://github.com/ShinHongKyu/spotify-my-history/tree/main/account

**Comment by ghost** [comment]
홍규님! 본인의 데이터를 분석해보고 계시군요~ 본인에 대해 더 잘 알아가는 기회가 되실 것 같습니다.
취향, 습관 등은 주관적인 부분이라서 어떻게 객관성을 확보할 수 있을지 고민해보시면 좋을 것 같네요!
